features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsView.kt

```diff
@@ -332,10 +335,17 @@ private fun RoomHeaderSection(
             .padding(horizontal = 16.dp),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Avatar(
+        CompositeAvatar(
             avatarData = AvatarData(roomId.value, roomName, avatarUrl, AvatarSize.RoomHeader),
+            heroes = heroes.map { user ->
+                AvatarData(
+                    id = user.userId.value,
+                    name = user.displayName,
+                    url = user.avatarUrl,
+                    size = AvatarSize.RoomHeader
+                )
+            },
             modifier = Modifier
-                .size(70.dp)
```

Comment:
This removal is actually fixing a rendering issue, see the recorded screenshot. The Avatar was not rendered correctly.